packages/grpc-js-core/src/channel.ts

```diff
@@ -150,7 +153,7 @@ export class Http2Channel extends EventEmitter implements Channel {
           return checkServerIdentity(sslTargetNameOverride, cert);
         }
       }
-      subChannel = http2.connect(this.authority, connectionOptions);
+      subChannel = http2.connect(this.authority, connectionOptions) as typeof subChannel;
```

Comment:
What's up with this change?

Reply:
Artifact of experimenting earlier... removing.